### ISSUE TITLE
Support ALC from https://gcutils.de/lab2gpx/

### ISSUE THREAD

**Issue by capoaira** [cgeo/cgeo#10558]
## What function are you missing

If I use the download from https://gcutils.de/lab2gpx/ the Labs are shown as Virtuell Caches and not as labs.

## Suggested solution

It would be nice if labs from lab2gpx were also shown as labs.

**Comment by jakedot** [comment]
Are the Lab Cache type Geocaches in the GPX Adventures with the stage Lab Caches as waypoints or are the individual stage Lab Caches exported as a geocache each?

**Comment by capoaira** [comment]
> Are the Lab Cache type Geocaches in the GPX Adventures with the stage Lab Caches as waypoints or are the individual stage Lab Caches exported as a geocache each?

Each station is a geocache. The Adventure location is not included

**Comment by jakedot** [comment]
So they are not the same thing as our LC Connector caches. I guess they should not be parsed to be displayed like adventures then, it is confusing. See  #10479 for context

**Comment by fm-sys** [comment]
I think, it is even more confusing if they are parsed as virtual cache. We only read the gpx file. Whether https://gcutils.de/lab2gpx/ stores stages as caches or waypoints is IMHO their issue. We had implemented an explicit routing of labs to virtual caches in 2014 as there was no lab cache type inside c:geo which is nowadays definitely no longer valid. If users want to display imported lab stages as individual caches they can still disable the ALC connector to avoid duplications...

**Comment by jakedot** [comment]
well we could have two different cache types, one for the adventures and one for the lab caches. That way, there's no confusion as well. And if we decide to parse lab caches out of the connector one day, we already have the type defined.

**Comment by capoaira** [comment]
Tow types of Labs are very confusing:
You always say I search a lab cache, you never say I search an Adventure.
Most times Lab Cache and ALC means the same.

**Comment by baiti** [comment]
From my perspective it is pretty much like a Multi except that "Stages" increment the "Found it" statitics too. However, the Adventure itself is marked complete only if you have all "Stages" completed. Geocaching.com maintains two counters.
1) Completed (finished) Adventures
2) A count of all Stages you have ever solved, independent to which Adventures they might  belong to.

In the regular stats, you see (2). I haven't seen (1) explicity shown, but it exists if you use the public API:
```
GET /Api/Accounts/GetAccount
```

You will receive:
```
{
  "Id": 0,
  "UserName": "string",
  "PublicGuid": "00000000-0000-0000-0000-000000000000",
  "Avatar": {
    "AvatarUrl": "string"
  },
  "GeocacheFindCount": 0,
  "CompletedAdventures": 0,
  "MembershipLevel": 0,
  "SignupDate": 0,
  "MembershipExpiryDate": 0,
  "HideCount": 0
}
```
Look for `CompletedAdventures` it is what I have described above as (1). What's really confusing is that `GeocachFindCount` is the total you have found, including all found Stages from Adventures excluding the `CompletedAdventures`.

I have yet to talk to a single Geocacher who understands and supports that strange idea of what should count a a "Found it".

**Comment by fm-sys** [comment]
I think we can ask https://gcutils.de if they are willing to implement a feature to export lab stages as waypoints...

**Comment by fm-sys** [comment]
Fixed in PR #10570
